node-server-sdk.api.md

```diff
@@ -4,23 +4,26 @@
 
 ```ts
 
-// @public
-export function getInstance(): IEppoClient;
-
-// @public
-export interface IAssignmentEvent {
-    experiment: string;
-    subject: string;
+import { EppoClient } from '@eppo/js-client-sdk-common';
+import { IAssignmentEvent } from '@eppo/js-client-sdk-common';
+import { IAssignmentLogger } from '@eppo/js-client-sdk-common';
+import { IConfigurationStore } from '@eppo/js-client-sdk-common';
+import { IEppoClient } from '@eppo/js-client-sdk-common';
+
+// @public (undocumented)
+export class EppoServerClient extends EppoClient implements IEppoClient {
+    // Warning: (ae-forgotten-export) The symbol "IPoller" needs to be exported by the entry point index.d.ts
+    constructor(configurationStore: IConfigurationStore, poller: IPoller);
     // (undocumented)
-    subjectAttributes: Record<string, any>;
-    timestamp: string;
-    variation: string;
+    stopPolling(): void;

```

Comment:
This is the one method in the server-side Eppo client that isn't in the other SDKs. Since it isn't obvious if this is actually in use or not, I extended `EppoClient` so we could keep it in. Although it would be nice to take it so we could be more consistent among the SDKs, I also don't want to risk the off chance that any customers were depending on this function being available

Reply:
I have a hunch that if this is used, its used in the context of automated tests so that they exit normally/gracefully. That would explain other SDK's not having it as as a node unit testing specific problem. 

But it's just a hunch.

Reply:
It was introduced for tests

Reply:
Ah, thanks for the context @ploomis14 !

Yeah I can't imagine any scenario where anyone would want to use this in production. Stopping polling means the SDK is no longer in sync with their assignment configuration, which is a risky scenario to be in. Usage should be discouraged if not removed